app/forms/error_messages.py

```diff
@@ -3,14 +3,22 @@
 # Set up default error and warning messages
 # lazy_gettext is used as the string needs to be resolved at runtime
 # for translations to work.
+
+# NOTE: `%` style string formatting is used for compatibility with WTForms default field
+# validators - using `.format` for non-WTForm field validation would introduce
+# inconsistency.
 error_messages = {
     "MANDATORY_QUESTION": lazy_gettext("Enter an answer to continue"),
     "MANDATORY_TEXTFIELD": lazy_gettext("Enter an answer to continue"),
     "MANDATORY_NUMBER": lazy_gettext("Enter an answer to continue"),
     "MANDATORY_TEXTAREA": lazy_gettext("Enter an answer to continue"),
-    "MANDATORY_RADIO": lazy_gettext("Select an answer to continue"),
+    "MANDATORY_RADIO": lazy_gettext(
+        'Select an answer <span class="u-vh">to ‘%(question_title)s’</span>'
```

Comment:
Not sure if the card is correct or not, but it doesn't specify quoting the question text.

Reply:
Checked with Donna - question title is supposed to be quoted :+1: